pkg/model/model.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package model;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+message Traces {
+  repeated Trace traces = 1 [(gogoproto.nullable) = false];
+}
+
+message Trace {
+  uint64 trace_id = 1;
+  repeated Span spans = 2 [(gogoproto.nullable) = false];
+}
+
+message Span {
```

Comment:
interesting.. since the format is different it might be good to clarify your README statement

ex. "Internally Loki is really just an opinionated reimplementation of OpenZipkin."

Reimplementation typically uses the same model. Seems this is more like jaeger which has a different model, and supports interop to some degree. You might consider adding a section to the README explaining to users who find this tool what sort of interop is expected to work within zipkin. For example, it the api still compatible? if so which endpoints (is it just the collector or do the other endpoints still work)? If it is just collector, which of json or thrift work? If neither, then no existing zipkin instrumentation would work. On that note, since B3 isn't used, the tracer side of this isn't a reimplementation rather (again similar to jaeger) a different format and library.

Maybe least confusing is to stick with the first part of the README "inspired by" if interop isn't a goal.